src/OpenTelemetry.Exporter.Jaeger/README.md

```diff
@@ -64,25 +64,28 @@ properties:
 
 * `Protocol`: The protocol to use. The default value is `UdpCompactThrift`.
 
-  | Protocol       | Description                                           |
-  |----------------|-------------------------------------------------------|
-  |UdpCompactThrift| Apache Thrift compact over UDP to a Jaeger Agent.     |
-  |HttpBinaryThrift| Apache Thrift binary over HTTP to a Jaeger Collector. |
+  | Protocol         | Description                                           |
+  |------------------|-------------------------------------------------------|
+  |`UdpCompactThrift`| Apache Thrift compact over UDP to a Jaeger Agent.     |
+  |`HttpBinaryThrift`| Apache Thrift binary over HTTP to a Jaeger Collector. |
 
 See the [`TestJaegerExporter.cs`](../../examples/Console/TestJaegerExporter.cs)
 for an example of how to use the exporter.
 
 ## Environment Variables
 
 The following environment variables can be used to override the default
-values of the `JaegerExporterOptions`.
-
-| Environment variable               | `JaegerExporterOptions` property |
-| ---------------------------------- | -------------------------------- |
-| `OTEL_EXPORTER_JAEGER_AGENT_HOST`  | `AgentHost`                      |
-| `OTEL_EXPORTER_JAEGER_AGENT_PORT`  | `AgentPort`                      |
-| `OTEL_EXPORTER_JAEGER_ENDPOINT`    | `Endpoint`                       |
-| `OTEL_EXPORTER_JAEGER_PROTOCOL`    | `Protocol`                       |
+values of the `JaegerExporterOptions`
+(following the [OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#jaeger-exporter)).
+
+  <!-- markdownlint-disable MD013 -->
```

Comment:
minor comment:
It can be configured globally like this: https://github.com/signalfx/signalfx-dotnet-tracing/blob/0a98256a339210ba08e7ebfea94367ae8627d9a0/docs/.markdownlint.yaml#L1-L7

Reply:
Sure, IMO not this PR. Current approach is common for whole repository.

Reply:
See #2917